docs/zh_cn/tutorials/config.md

```diff
@@ -186,7 +186,7 @@ model = dict(
         rpn_proposal=dict(  # 在训练期间生成 proposals 的配置
             nms_across_levels=False,  # 是否对跨层的 box 做 NMS。仅适用于 `GARPNHead` ，naive rpn 不支持 nms cross levels。
             nms_pre=2000,  # NMS 前的 box 数
-            nms_post=1000,  # NMS 要保留的 box 的数量，只在 GARPNHHead 中起作用。
```

Comment:
Same question in both English and Chinese documents. Please fix English Docs too.

Reply:
The English documentation looks good, right?